CMakeModules/BuildErlang.cmake

```diff
@@ -66,7 +66,10 @@ macro(pack_lib avm_name)
         COMMENT "Packing runnable ${avm_name}.avm"
         VERBATIM
     )
-    add_dependencies(${avm_name} ${avm_name}_main ${ARCHIVE_TARGETS} PackBEAM)
+    add_dependencies(${avm_name} ${ARCHIVE_TARGETS} PackBEAM)
```

Comment:
Is this change needed because of the code coverage targets?  I am fine with it -- just curious why.

Reply:
Actually no, this is a fix for a CMake warning. In one case ${avm_name}_main doesn't exist, hence the block below.

Reply:
👍🏽